usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -572,6 +574,7 @@ def set_if_found(dl_name, path, model, file):
             'recipient_parent_name': 'contract_data__ultimate_parent_legal_enti',
             'recipient_phone_number': 'contract_data__vendor_phone_number',
             'recipient_state_code': 'contract_data__legal_entity_state_code',
+            'recipient_state_name': 'contract_data__legal_entity_state_descrip',
```

Comment:
Just something i noticed is that `TransactionNormalized` has `recipient` and you could go down an alternate route to get the same data here through TransactionNormalized -> Recipient -> Location. This would cause additional joins, so this current way is still probably better performance-wise